src/main/plugin/iso19115-3.2018/schematron/schematron-rules-inspire.sch

```diff
@@ -616,8 +616,10 @@
 			<sch:let name="accessConstraints" value="string-join(mco:accessConstraints/*/@codeListValue, ', ')"/>
 			<sch:let name="classification" value="string-join(mco:classification/*/@codeListValue, ', ')"/>
 			<sch:let name="otherConstraints" value="mco:otherConstraints/gco:CharacterString/text()"/>
-			<sch:let name="useLimitation" value="mco:useLimitation/*/text()"/>
-			<sch:let name="useLimitation_count" value="count(mco:useLimitation/*/text())"/>
+			<!-- <sch:let name="useLimitation" value="mco:useLimitation/*/text()"/> -->
```

Comment:
You should remove unused code comment.

Reply:
I was not removing comments only for the sake of clarity in this intermediate phase. Next time I will avoid. Sorry for this mistake.